docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/rdb-loader-configuration-reference/index.md

```diff
@@ -27,11 +27,14 @@ This is a complete list of the options that can be configured:
 | `port`                     | Required. Port of Redshift cluster.                                                                                                                                             |
 | `database`                 | Required. Redshift database which the data will be loaded to.                                                                                                                   |
-| `roleArn`                  | Required. AWS Role ARN allowing Redshift to load data from S3.                                                                                                                  |
+| `roleArn`                  | Required if 'NoCreds' is chosen as load auth method.. AWS Role ARN allowing Redshift to load data from S3.                                                                                                                  |
 | `schema`                   | Required. Redshift schema name, eg “atomic”.                                                                                                                                    |
 | `username`                 | Required. DB user with permissions to load data.                                                                                                                                |
 | `password`                 | Required. Password of DB user.                                                                                                                                                  |
 | `maxError`                 | Optional. Configures the [Redshift MAXERROR load option](https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-data-load.html#copy-maxerror). The default is 10.       |
+| `loadAuthMethod.*`         | Optional, default method is `NoCreds`. Specifies the auth method to use with 'COPY' statement. |
+| `loadAuthMethod.type`      | Required if `loadAuthMethod` section is included. Specifies the type of the auth method. The possible values are `NoCreds` and `TempCreds`. <br/><br/>With 'NoCreds', no credentials will be passed to 'COPY' statement. Instead, Redshift cluster needs to be configured AWS Role ARN allowing Redshift to load data from S3 and this Role ARN needs to be passed field 'roleArn' above. More information can be found [here](https://docs.aws.amazon.com/redshift/latest/dg/copy-usage_notes-access-permissions.html). <br/><br/>With 'TempCreds', temporary credentials will be created for every load operation and these temporary credentials will be passed to 'COPY' statement. |
+| `loadAuthMethod.roleArn`   | Required if `loadAuthMethod.type` is `TempCreds`.IAM role that is used while creating temporary credentials. Created credentials will allow to access resources specified in the given role. “s3:GetObject*”, “s3:ListBucket”, and “s3:GetBucketLocation” permissions for transformer output S3 bucket should be specified in the role.   
```

Comment:
I find it very confusing that we have two different `roleArn` settings...

Reply:
I agree but there is not much thing to do. Default method is `NoCreds`. Maybe it is best to not document `TempCreds` at all. As I said in the previous comment, for ordinary users, no method has any significant advantage to one to another. New `TempCreds` method is slightly easier to configure but there is not much difference.